LayoutTests/interaction-region/layer-tree-expected.txt

```diff
@@ -0,0 +1,132 @@
+
```

Comment:
Note: this test is more deterministic than the others, because the remote layer tree is ordered.
If the `UIHelper.getCALayerTree()` thing makes sense, we could upgrade the other tests too in a separate patch.